pytorch_lightning/utilities/apply_func.py

```diff
@@ -25,7 +26,11 @@
 from pytorch_lightning.utilities.imports import _TORCHTEXT_AVAILABLE
 
 if _TORCHTEXT_AVAILABLE:
-    from torchtext.data import Batch
+    from torchtext import __version__ as torchtext_version
```

Comment:
can we use `distutils.version.LooseVersion` here?? It's been used at other places too in the codebase. You can check.

Reply:
Thanks @rohitgr7 My fourth commit uses LooseVersion

Reply:
you could also check `_module_available("torchtext.legacy")`, maybe a bit cleaner than coding the version commit.

Reply:
Hi @awaelchli 
I have just made a commit using _module_available as you suggested